### ISSUE TITLE
Server related questions

### ISSUE THREAD

**Issue by SeongJongKwak** [256dpi/processing-mqtt#32]
Server related questions

  hello. I don't know, but I want to ask a question.

Server: https://www.shiftr.io/try.
is this run by you?
Or where do you operate from?

Can I use it for free for life? I'm asking because I don't know.

**Comment by 256dpi** [comment]
Yes, shiftr.io is run by my company. You may use shiftr.io as you like, we hope to keep the current offering as long as possible.
